### ISSUE TITLE
Enhance double-checking feature

### ISSUE THREAD

**Issue by sarahstrong314** [thewca/wca-live#27]
It would be excellent if admins could click a checkbox to show that a scorecard has been double checked. It would be even better if the results that showed under the double-checking feature could be filtered to show only the scorecards that have not been double checked.

**Comment by Ranzha** [comment]
Agree, with a keyboard shortcut! Maybe +/=? That way, it's accessible near number keys regardless if scoretakers have a numpad.

Intended functionality could be that +/= or a button toggled whether the card is un-double-checked/double-checked. Also, the particular user who double-checked a card would be recorded and displayed to admin-side.

**Comment by AlexKatyukov** [comment]
And another idea for double-checking. Sometimes all cards are messed. It will be good to select all symbols in the input field when clicked on it to insert new id.

**Comment by Nanush7** [comment]
I think this is a good idea. I would also like to see something like a green tick next to the already double-checked rounds in the "Events" panel.
